src/components/Column/Column.js

```diff
@@ -8,47 +8,54 @@ import * as PropTypes from 'prop-types';
 import CardItem from '../CardItem/CardItem';
 import CardForm from '../CardForm/CardForm';
 
-const Column = ({ cards, addNewCardToBoard }) => {
+const Column = ({ columnProp, addNewCardToBoard }) => {
   const [cardFormEdit, setCardFormEdit] = useState(false);
-
-  const renderFooter = () => (
-    <div style={{ marginTop: '15px' }}>
-      {cardFormEdit
-        ? (
-          <CardForm handleCancelButton={setCardFormEdit} handleAddCard={addNewCardToBoard} />
-        )
-        : (
-          <Button
-            onClick={() => setCardFormEdit(!cardFormEdit)}
-            style={{ textTransform: 'capitalize' }}
-            size="medium"
-            startIcon={<AddIcon />}
-          >
-            Add a card
-          </Button>
-        )}
-    </div>
-  );
-
+  const passNewCardToBoard = (newCard) => {
```

Comment:
You don't need this function, you can just pass to the `CardForm` component the property you just received called `addNewCardToBoard`. This function is just delegating so you can just send that as prop.

Reply:
removed and passed function directly